### PR TITLE
[GITHUB] Update Labeler action to v4

### PR DESCRIPTION
**Pull request by sarthakroy2002**

### CODE REVIEW

**Comment by SergeGautherie** [comment]
This should be a good idea.
Review changes at https://github.com/actions/labeler/releases :
for example, 'Make the repo-token input optional. Default is github.token'.

**Comment by sarthakroy2002** [comment]
> This should be a good idea. Review changes at https://github.com/actions/labeler/releases : for example, 'Make the repo-token input optional. Default is github.token'.

I don't think any changes are needed to switch from v3 to v4.

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v3
+    - uses: actions/labeler@v4
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (SergeGautherie) [inline .github/workflows/labeler.yml:11]: Would it still work with the following removal?
```suggestion
```

Comment (sarthakroy2002) [reply .github/workflows/labeler.yml:11]: Keeping it won't cause any issue

**Pull request closed without merging**